README.md

```diff
@@ -1,6 +1,32 @@
 # Computational Crystallography Toolbox
 
-## Introduction
+Greetings, earthling. We see you intend to embark on the arduous quest of becoming an elite computational crystallographer. Many before you have dared to try. We hope the CCTBX package grants you success along your journey. 
```

Comment:
@dermen while I enjoy the D&D / Zork (or whatever, not sure I get the reference) theming I am not sure that the enjoyment would be universal? 😉 have experience of people adding "funnies" which later on causes more annoyance than it was worth (DIALS `HalRaiser` I am looking at _you_)